samples/word/01-basics/basic-api-call.yaml

```diff
@@ -1,7 +1,6 @@
-order: 1
-id: word-basic-api-call
+id: word-basics-basic-api-call-
 name: Basic API call
-description: Executes a basic Word API call
+description: Colors the selected text in red.
```

Comment:
The reason for keeping it generic is so that when folks click on this basic sample, they don't get a bunch of "colors the selected text in red" descriptions in their own snippets (we copy over the descriptions from imported snippets / samples)